tests/php/src/Admin/OptionsMenuTest.php

```diff
@@ -290,6 +290,13 @@ public function test_render_screen_for_admin_user() {
 			)
 		);
 
+		// Set current screen to be the options menu.
+		set_current_screen( $this->instance->screen_handle() );
+
+		// Set title to be used in the screen.
+		global $title;
+		$title = 'Test Title';
```

Comment:
I see this is coming from 1e8de9a582bea7d594f65ccd3e1fcff351aef945. It was needed due to a PHP warning during tests?

Reply:
Yes, while rendering the screen it requires the title, but the title is not being set anywhere.